RocketChat/Rocket.Chat
# fix: cleaning the room filter between navigation on the admin section

## Proposed changes (including videos or screenshots)

**Resolve Room Filter Persistence Issue with Local Storage**

- Addressed a bug where room filter settings were only partially resetting during admin page navigation.
- Now, all filters are completely cleared.

## Issue(s)


https://rocketchat.atlassian.net/browse/CORE-217

## Steps to test or reproduce


1. Go to admin page
2. Go to Room section
3. Change the filter
4. Navigate to another section
5. Go back to the Room filter, the filter should be cleaned

## Further comments


https://github.com/RocketChat/Rocket.Chat/assets/35040806/054ee962-1903-483b-bddd-c7f02189eff9
